contrib/cirrus/lib.sh

```diff
@@ -108,9 +109,10 @@ bigto() { err_retry 7 5670 "" "$@"; }  # 12 minutes max
 
 # Print shell-escaped variable=value pairs, one per line, based on
 # variable name matching a regex.  This is intended to support
-# passthrough of CI variables from host -> container or from root -> user.
-# For all other vars. we rely on tooling to load this library from inside
-# the container or as rootless user to pickup the remainder.
+# passthrough of variables from Cirrus-CI for several different
+# testing setup situations (like contianer and rootless). Otherwise
+# Cirrus-CI operations always run as root on the system, so this is
+# not necesary.
```

Comment:
note to me: get rid of this new comment addition, it's wrong and unnecessary (commented elsewhere).  Keep function description.